Firestore/Example/Tests/SpecTests/FSTSpecTests.mm

```diff
@@ -340,6 +352,17 @@ - (void)doUnlisten:(NSArray *)unlistenSpec {
   [self.driver removeUserListenerWithQuery:std::move(query)];
 }
 
+- (void)doLoadBundle:(NSString *)bundleJson {
+  const auto &database_info = [self.driver databaseInfo];
+  BundleSerializer bundle_serializer(remote::Serializer(database_info.database_id()));
+  auto data = util::MakeString(bundleJson);
+  auto bundle = absl::make_unique<util::ByteStreamCpp>(
+      absl::make_unique<std::stringstream>(std::stringstream(data)));
+  auto reader = std::make_shared<BundleReader>(std::move(bundle_serializer), std::move(bundle));
+  auto task = std::make_shared<LoadBundleTask>(user_executor_);
+  [self.driver loadBundleWithReader:reader task:task];
```

Comment:
Optional nit: you can move these variables.

Reply:
Done.